setup.py

```diff
@@ -27,8 +27,8 @@
     'numpy >=1.13.3',
     'Pillow',
     'psutil >=5.7.3',
-    'python-dotenv'
-    
+    'python-dotenv',
+    'boto3'
```

Comment:
Let's remove this from setup.py. A lot of zm users don't want google or amazon packages installed automatically (privacy of data, I know, opinions differ here). I personally don't mind it, but there are several who raise concerns in this community. Are you aware of a conditional install for setup.py? It would be useful.

Reply:
It's just a package. Without credentials it won't do anything. It's about as dangerous as the 'requests' module, that too can call the AWS API ;)

Reply:
Anyway, done.